db/schema.rb

```diff
@@ -295,7 +295,6 @@
     t.uuid "patient_business_identifier_id", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["patient_business_identifier_id"], name: "index_passport_auth_on_business_id"
```

Comment:
This is a recent model we added. Our migrations on `master` don't show us adding this index. The addition of the index appears to be a mistake I made [here](https://github.com/simpledotorg/simple-server/commit/b3f7aff519d7c3690e530abd9d8095aa35a60a89). It's correct for it to be removed.

Now, if we _do_ want to index this field, we can add a separate migration to do so :)